src/e2e-test/resources/testdata/GCS_WILDCARD_TEST/test/wildcardFile1.csv

```diff
@@ -0,0 +1,9 @@
+id,name,yearofbirth,tablename,offset
```

Comment:
do we need that much data ? for the csv file ? it's just for test deletion , right ? empty or even 1 character file will be enough , right ?

Reply:
yeah, will use the empty file